target/linux/ramips/image/mt7621.mk

```diff
@@ -518,6 +518,15 @@ define Device/youhua_wr1200js
 endef
 TARGET_DEVICES += youhua_wr1200js
 
+define Device/youku-yk2
+  DTS := YOUKU-YK2
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  DEVICE_TITLE := YOUKU YK2
```

Comment:
I think it should be renamed to Youku YK-L2 as well.

Reply:
ok , will change.